public/assets/js/ability-scores.js

```diff
@@ -0,0 +1,115 @@
+// Retrieve selected race ability bonus info
```

Comment:
I pulled this from the branch @NicaVulcan is working on knowing it will change, but I didn't want the app to be partially working.